kornia/augmentation/augmentation.py

```diff
@@ -543,6 +544,10 @@ class CenterCrop(AugmentationBase2D):
             applied to each. Default: False.
         keepdim (bool): whether to keep the output shape the same as input (True) or broadcast it
                         to the batch form (False). Default: False.
+        mode (str): The used algorithm to crop. ``slice`` will use advanced slicing to extract the tensor based
```

Comment:
Let's try another name. mode is taken in other functions as in affine for bilinear or nearest. Might have some confusions there. Probably like how or something.

Reply:
I'll add for now `crop_mode`